src/index.ts

```diff
@@ -66,7 +67,7 @@ function getExtendConfigPath(options: N9ConfOptions, confPath: string): string {
 }
 
 function mergeWithStrategy(
-	strategy: N9ConfMergeStrategy = N9ConfMergeStrategy.V2,
+	strategy: N9ConfMergeStrategy = N9ConfMergeStrategy.V2, // eslint-disable-line
```

Comment:
this function is private, you can safely reorder the parameters to move optional onas at the end